src/i18n/i18n_module_es.properties

```diff
@@ -1,13 +1,13 @@
 actions=Acciones
 assign_all=Assignar todos
-assignGroups=Asignar a grupos
-assignRoles=Asignar a roles
-assignToOrgUnitsCapture=Asignar a unidades organizativas
-assignToOrgUnitsOutput=Asignar a unidades organizativas de salida
-replicateUser=Replicar usuario
+assign_groups=Asignar a grupos
+assign_roles=Asignar a roles
+assign_to_org_units_capture=Asignar a la captura de unidades organizativas
+assign_to_org_units_output=Asignar a la salida de las unidades de la organización
```

Comment:
Not sure about "unidades de la organización", we have "unidades organizativas" in the previous translation.